services/task/migrate.go

```diff
@@ -49,9 +49,9 @@ func runMigrateTask(ctx context.Context, t *admin_model.Task) (err error) {
 		}
 
 		if err == nil {
-			err = admin_model.FinishMigrateTask(ctx, t)
+			err = admin_model.FinishMigrateTask(db.DefaultContext, t)
```

Comment:
Hmm…
This does not sound like a good idea.
I mean, I understand that `ctx` is probably canceled after something like 5 seconds, but still, there must be another way of dealing with it…

Reply:
I know, but I don't have a good idea now.

Reply:
Because ctx is canceled here, so we cannot reuse it.

Reply:
> Hmm… This does not sound like a good idea. I mean, I understand that `ctx` is probably canceled after something like 5 seconds, but still, there must be another way of dealing with it…

We can find it in another PR but at least fix the bug in this PR.